### PR TITLE
[Non-modular] Vatbeast and oozing grapes don't leave their corpse behind

### PR DESCRIPTION
**Pull request by desertmote**
## About The Pull Request

Instead of having a bulky corpse that does nothing but sit in the hallways, they now drop 'unique' food items and instead leave a more fun mess to clean.

## Why It's Good For The Game

They just sat in the hallway, couldn't be disposal chuted, butchered, burned, juiced, or nothin'.

## Changelog
:cl:
expansion: Vatbeast mobs and the sholean grape type of ooze mobs now del their corpse upon death and drop loot.
/:cl:

### CODE REVIEW

File: modular_skyrat/modules/customization/modules/food_and_drinks/food/misc.dm [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+//cytology mobs butcher results
+/obj/item/food/grown/grapes/ooze
+	seed = /obj/item/seeds/grape/green
+	name = "bunch of oozing grapes"
+	icon_state = "greengrapes"
+	bite_consumption_mod = 3
+	tastes = list("gelatinous grape" = 1)
+	distill_reagent = /datum/reagent/toxin/slimejelly
+
+/obj/item/food/octopusrings
```
Comment (Gandalf2k15) [inline modular_skyrat/modules/customization/modules/food_and_drinks/food/misc.dm:12]: ewww

**Review by Gandalf2k15 — APPROVED** [review]

File: modular_skyrat/modules/customization/modules/food_and_drinks/food/misc.dm (hunk 1 above)
Comment (desertmote) [reply modular_skyrat/modules/customization/modules/food_and_drinks/food/misc.dm:12]: im not good at balancing so be honest if its too much

**Comment by Cheshify** [comment]
It's a nonmodular change that isn't even all that great. If we're changing upstream code, why not just go upstream and allow butchering?

**Comment by desertmote** [comment]
> It's a nonmodular change that isn't even all that great. If we're changing upstream code, why not just go upstream and allow butchering?

what isn't good about it? I've tried adding butchering, it's just one line. I didn't do it because it makes red blood and red gibs, of mobs that don't have blood or gibs. It's also not fun to clean an explosion of blood.

**Comment by Cheshify** [comment]
> what isn't good about it? I've tried adding butchering, it's just one line. I didn't do it because it makes red blood and red gibs, of mobs that don't have blood or gibs. It's also not fun to clean an explosion of blood.

I mean, a few corpses isn't the end of the world, and it's a fix that could be done upstream

**Comment by desertmote** [comment]
> > what isn't good about it? I've tried adding butchering, it's just one line. I didn't do it because it makes red blood and red gibs, of mobs that don't have blood or gibs. It's also not fun to clean an explosion of blood.
> 
> I mean, a few corpses isn't the end of the world, and it's a fix that could be done upstream

upstream doesn’t have maintenance mobs though, so it would be weird to add.

**Comment by desertmote** [comment]
i dont know what this is waiting for, besides ive since deleted the branch this pr was made in. just puttin' it out there.

**Pull request closed without merging**